documentation/staging/content/navigate/image.md

```diff
@@ -115,7 +115,8 @@ latest capabilities, many of which are exposed by this application.  As such, us
 recommended.
 
 {{% notice note %}} For "Model in Image" or "Domain in Image" [domain locations]({{< relref "/navigate/project-settings#choosing-a-domain-location" >}}),
-when building your primary or auxiliary image, we highly recommend using WDT 2.0 to take advantage of enhanced capabilities that reduce the chance of deploy-time errors created due to a bad model.    
+when building your primary or auxiliary image, we highly recommend using [WDT 2.0](https://github.com/oracle/weblogic-deploy-tooling/releases) to
+take advantage of enhanced capabilities that reduce the chance of deploy-time errors created due to a bad model.    
```

Comment:
Since 1.1.0 will embed WDT 2.0.x, we might want to make this statement stronger.  The reason being is that if they use Prepare Model, it may rewrite the model into a format that WDT 1.9.x cannot parse (e.g., if they have lists, WDT 2.0 write them using the hyphen-demarcation list syntax.  WDT 1.9.x cannot parse this...